test/e2e/tests/multichain/asset-list.spec.ts

```diff
@@ -146,20 +148,20 @@ describe('Multichain Asset List', function (this: Suite) {
         const assetListPage = new AssetListPage(driver);
         await headerNavbar.clickSwitchNetworkDropDown();
         await selectNetworkDialog.selectNetworkName(NETWORK_NAME_MAINNET);
-        await assetListPage.waitUntilAssetListHasItems(2);
+        await assetListPage.waitUntilAssetListHasItems(4);
         await assetListPage.clickOnAsset('TST');
         await driver.clickElement('.mm-box > button:nth-of-type(3)');
         const toastTextElement = await driver.findElement('.toast-text');
         const toastText = await toastTextElement.getText();
         assert.equal(
           toastText,
-          `You're now using ${LOCALHOST}`,
+          `You're now using ${POLYGON_NAME_MAINNET}`,
           'Toast text is correct',
         );
       },
     );
   });
-  it('shows correct asset and balance when swapping on a different chain', async function () {
+  it.only('shows correct asset and balance when swapping on a different chain', async function () {
```

Comment:
Can we remove this `only`

Reply:
done [28ae946](https://github.com/MetaMask/metamask-extension/pull/28822/commits/28ae94600bfcfee55c5f970b1c2f9ecc28ad5cb7)